series/series.rb

```diff
@@ -0,0 +1,29 @@
+class Series
+  def initialize(num)
+    @number=num.to_i
```

Comment:
@neelgohel Spacing issues. `@number=num.to_i` should looks like `@number = num.to_i`